docs/references/ayx-plugin-cli.md

```diff
@@ -417,6 +443,9 @@ to define these items:
     versions of a tool so that users can choose to upgrade to a new
     version or use the previous one inside of any Alteryx workflows.
 
+:information_source:
+Use `--omit-ui` argument if this plugin will not have UI. UI can be generated later by using the [generate-ui](#generate-ui) command.
```

Comment:
TW review:

Use the `--omit-ui` argument if this plugin doesn't have a UI. You can generate the UI later with the [generate-ui](#generate-ui) command.